draft-ietf-ppm-dap.md

```diff
@@ -1099,84 +1099,170 @@ refinement process. We instead think of these as properties of the output
-  N2 | AggregationJobContinueReq                           |
-  P2 | prep_msg_2, [leader_prep_share_3]                   |
-[N3] |---------------------------------------------------->| P2
-     |                                  AggregationJobResp | [N3]
-     |                 [prep_msg_3, {helper_prep_share_4}] | [P3]
-  P3 |<----------------------------------------------------| {N4}
-  N4 | AggregationJobContinueReq                           |
-  P4 | prep_msg_4, [leader_prep_share_5]                   |
-[N5] |---------------------------------------------------->| P4
-     |                                  AggregationJobResp | [N5]
-     |                 [prep_msg_5, {helper_prep_share_6}] | [P5]
-     |<----------------------------------------------------| {N6}
-     |                                                     |
-    ...                                                   ...
-Pr-2 |                                                     |
-Nr-1 | AggregationJobContinueReq                           |
-Pr-1 | prep_msg_r-1, [leader_prep_share_r]                 |
-[Nr] |---------------------------------------------------->| Pr-1
-     |                                  AggregationJobResp | [Nr]
-     |                               [helper_prep_share_r] | [Pr]
-   F |<----------------------------------------------------| F
-     |                                                     |
-     v                                                     v
-    leader_out_share                         helper_out_share
-
-      [] - Indicates a field that is only sent if the VDAF
-           requires at least one more round.
-      {} - Indicates a field that is only sent if the VDAF
-           requires at least two more rounds.
-       I - Produce first-round prep-share.
-       P - Combine prep shares into prep message.
-       N - Produce next prep share.
-       F - Produce output share.
-~~~
-{: #agg-flow title="Mapping of VDAF preparation to the DAP aggregation sub-protocol."}
-
-The exact protocol structure depends on the number of rounds of preparation
-that are required by the VDAF. (The number of rounds is a constant parameter
-specified by the VDAF itself: Prio3 has just one round, but Poplar1 requires
-two.) The high-level idea is that the Aggregators take turns running the
-computation locally until input from their peer is required:
+the Helper's report share for each report in the job, and for each report the
+initialization step for preparation. The Helper's response, along with each
+subsequent request and response, carries the remaining messages exchanged
+during preparation.
+
+~~~ ladder
+  report, agg_param
+   |
+   v
++--------+                                         +--------+
+| Leader |                                         | Helper |
++--------+                                         +--------+
+   | AggregationJobInitReq                               |
+   | agg_param, report_share, prep_step(initialize)      |
```

Comment:
Is this saying:
"Send an AggregationJobInitReq consisting of[agg_param, report_share, and the output of prep_step()]"

If so, maybe
```suggestion
   | AggregationJobInitReq:                           |
   |    agg_param, report_share, prep_step(initialize)      |
```
e

Reply:
That's right. Done.